pkg/nodejs/packagejson/parse.go

```diff
@@ -35,18 +42,24 @@ func (p *Parser) Parse(r io.Reader) (Package, error) {
 		return Package{}, xerrors.Errorf("JSON decode error: %w", err)
 	}
 
+	license := parseLicense(pkgJSON.License)
+	if !pkgJSON.hasContent(license) {
+		return Package{}, nil
+	}
```

Comment:
As @afdesk said, I don't think we need this logic. Returning a license only seems to be useless.
https://github.com/aquasecurity/go-dep-parser/pull/231#discussion_r1234800190